recipes/samarium/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2022 Jai Bellare
+# See <https://opensource.org/licenses/MIT/> or LICENSE.md
+# Project homepage: <https://github.com/strangeQuark1041/samarium>
+
```

Comment:
```suggestion
```
The project license is listed in LICENSE file. Also, recipes here are under Conan-io "property", but all them are MIT license.